### ISSUE TITLE
[Bug]: error with installing dependencies for vue

### ISSUE THREAD

**Issue by czarsbenemoriendi** [vuestorefront/storefront-ui#2793]
### Describe the Bug

Hi!

There was an error while installing the dependencies for `@storefront-ui/vue.` This package cannot be installed using npm and pnpm, but it was successfully installed when I used Yarn.

1. here is error by pnpm
```
pnpm add -D @nuxtjs/tailwindcss @storefront-ui/vue   
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@storefront-ui/tw-plugin-peer-next@workspace:*" is in the dependencies but no package named "@storefront-ui/tw-plugin-peer-next" is present in the workspace
```
2. and error by npm
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in:
```
Here is link to stackblitz: https://stackblitz.com/edit/nuxt-starter-xnjcby?file=README.md

### Expected behavior

These packages should be installed.

### SFUI version

@storefront-ui/vue@2.3.0

### Framework

Nuxt

### Node version

v19.0.0

### Browser

does not apply

### OS

macOS

### Able to fix / change the documentation?

- [ ] Yes
- [ ] No

### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

**Comment by FRSgit** [comment]
Hey @czarsbenemoriendi,

Thanks for filing this issue!

It seems that something it wrong with this release `@storefront-ui/tailwind-config` package indeed. We're investigating the issue - I will come back to you when it's solved 🤞

**Comment by FRSgit** [comment]
Hey @czarsbenemoriendi,

Closing this issue - it's fixed in `2.3.1` - please file a new issue if you think that this error still occurs
